### DISCUSSION TITLE
Not able to pull large datasets, tool hangs on running.

### DISCUSSION THREAD

**Discussion by kratintiwari** [projectdiscovery/cvemap#56] [Q&A]
### Current Behavior: 
When I try to run this command 
`cvemap -poc=true -kev=false -epk -field cwe,product,poc -json > cvemap_output.json`
It just keeps running and only deliver 99 results.

### Expected Behavior: 
It should give me all the expected as shown in the terminal which is **96288** cve records in json.

### Steps To Reproduce:

Example: steps to reproduce the behavior:
1. Run `cvemap -poc=true -kev=false -epk -field cwe,product,poc -json > cvemap_output.json`
2. It would fetch the results but it keeps running

### Anything else:
I can't describe with a screenshot as the tool works fine although the desired result set is not being delivered by the tool.

**Comment by ehsandeep** [comment]
@kratintiwari you are using interactive mode by using `-epk` option that requires you to manually navigate the results, it's not hanging, instead you are using interactive mode. 

```
   -epk, -enable-page-keys  enable page keys to navigate results
```

about result count, as default 50 results are pulled from api and can be increased upto 100 per response, later you can use `-epk` option to navigate the results for further pages. 

```
   -l, -limit int  limit the number of results to display (default 50)
```

Just to mention, this CLI helps you navigate through CVE data and not dump all the data at once.

**Reply by kratintiwari** [reply]
@ehsandeep 
Thank for the response. 
Although my ask is the same, if I want to curate all of the json data for CVEs that have poc and are not in CISA KEV in a single file, can I do that with this tool ?

**Reply by ehsandeep** [reply]
it's not possible at the moment, but should be possible to support in future updates.

**Reply by kratintiwari** [reply]
Is there a work around that you could suggest to get that type of data?

Thanks again for clarifying.

**Reply by kratintiwari** [reply]
Also can I get the poc url in interactive form (-epk) ? I can see the `poc `object has attribute `url` which is displayed in json although wanted to know if I can see that in the tabular format when we use `-epk` 

Thanks
